src/pages/instances/InstanceSnapshots.tsx

```diff
@@ -71,46 +113,107 @@ const InstanceSnapshots: FC<Props> = ({ controlTarget, instance }) => {
     };
   });
 
+  const {
+    pageData: pageSnapshots,
+    currentPage,
+    paginate: setCurrentPage,
+  } = usePagination(rows, {
```

Comment:
We could create an adapter for `usePagination`, that also holds the `pageSize`. We currently need to define the latter separately every time (also in `InstanceList`). I think we will always want a `pageSize` when we use pagination, so it might be good to combine them. I think we should add this as a small story to the backlog to not extend the scope of this further.

Reply:
[Done](https://warthogs.atlassian.net/browse/WD-2713)